app/helpers/podcasts_helper.rb

```diff
@@ -19,10 +19,14 @@ def first_nonblank(type, items)
   end
 
   def show_itunes_summary?(model)
-    model.summary || model.description
+    model.summary.present? || model.description.present?
```

Comment:
It seemed like an oversight not to check for empty strings `""` in all these summary/description checkers?  So added them both here (in the RSS generation) and in the API.

Reply:
great!